xarray/core/common.py

```diff
@@ -926,6 +926,14 @@ def rolling_exp(
         --------
         core.rolling_exp.RollingExp
         """
+
+        if "keep_attrs" in window_kwargs:
```

Comment:
I am not sure if I should warn here. Nobody ever complained. However, all other functions either warn (from a deprection) or error (because `keep_atttrs` is not a valid keyword).